src/main/java/roomescape/service/ThemeService.java

```diff
@@ -0,0 +1,47 @@
+package roomescape.service;
+
+import org.springframework.stereotype.Service;
+import roomescape.domain.Theme;
+import roomescape.dto.theme.ThemeResponse;
+import roomescape.dto.theme.create.ThemeCreateRequest;
+import roomescape.dto.theme.create.ThemeCreateResponse;
+import roomescape.exception.custom.DuplicatedThemeNameException;
+import roomescape.repository.ThemeRepository;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Service
+public class ThemeService {
+
+    private final ThemeRepository themeRepository;
+
+    public ThemeService(ThemeRepository themeRepository) {
+        this.themeRepository = themeRepository;
+    }
+
+    public List<ThemeResponse> findThemes() {
+        return themeRepository.findThemes()
+                .stream()
+                .map(theme -> ThemeResponse.toDto(theme))
+                .collect(Collectors.toList());
```

Comment:
```suggestion
                .map(theme -> ThemeResponse.toDto(theme))
                .toList();
```
와 같이 수정할 수 있을 것 같아요!